src/Microsoft.OpenApi/Models/OpenApiReference.cs

```diff
@@ -18,23 +18,23 @@ public class OpenApiReference : IOpenApiSerializable, IOpenApiDescribedElement,
         /// A short summary which by default SHOULD override that of the referenced component.
         /// If the referenced object-type does not allow a summary field, then this field has no effect.
         /// </summary>
-        public string Summary { get; set; }
+        public string? Summary { get; set; }
 
         /// <summary>
         /// A description which by default SHOULD override that of the referenced component.
         /// CommonMark syntax MAY be used for rich text representation.
         /// If the referenced object-type does not allow a description field, then this field has no effect.
         /// </summary>
-        public string Description { get; set; }
+        public string? Description { get; set; }
 
         /// <summary>
         /// External resource in the reference.
         /// It maybe:
         /// 1. a absolute/relative file path, for example:  ../commons/pet.json
         /// 2. a Url, for example: http://localhost/pet.json
         /// </summary>
-        public string ExternalResource { get; init; }
-
+        public string? ExternalResource { get; init; }
```

Comment:
leaving the comment here because I can't below.
I'm not sure Type should be nullable